app.js

```diff
@@ -1,193 +1,331 @@
-  },
-  cookieSales: function() {
-    return Math.round(this.custPerHr() * this.avgSale);
-  },
-  cookiesPerHour: function() {
-    for(var i = 0; i < 16; i++) {
-      if (i < 15){
-        var cookieSales = this.cookieSales();
-        this.totalDay += cookieSales;
-        this.saleResults.push(cookieSales);
-        var newLi = document.createElement('li');
-        var newContent = document.createTextNode(this.hours[i] + ': ' + cookieSales + ' cookies');
-        newLi.appendChild(newContent);
-        var current = document.getElementById('li3');
-        document.body.insertBefore(newLi, current);
-      } else {
-        var newLiZ = document.createElement('li');
-        var newContentZ = document.createTextNode('Total: ' + this.totalDay);
-        newLiZ.appendChild(newContentZ);
-        var currentZ = document.getElementById('li3');
-        document.body.insertBefore(newLiZ, currentZ);
-        break;
-      }
-    }
-  },
+
+stores[0].custPerHr();
+
+//cookie sales per hour
+Store.prototype.cookieSales = function() {
+  return Math.round(this.custPerHr() * this.avgSale);
 };
-console.log(seattleCenter);
-seattleCenter.cookiesPerHour ();
-
-var capitolHill = {
-  min : 20,
-  max : 38,
-  avgSale : 2.3,
-  saleResults: [],
-  totalDay:0,
-  hours : ['6am', '7am', '8am', '9am', '10am', '11am', '12pm', '1pm', '2pm', '3pm', '4pm', '5pm', '6pm', '7pm', '8pm'],
-  custPerHr: function() {
-    return Math.random() * (this.max - this.min) + this.min;
-  },
-  cookieSales: function() {
-    return Math.round(this.custPerHr() * this.avgSale);
-  },
-  cookiesPerHour: function() {
-    for(var i = 0; i < 16; i++) {
-      if (i < 15){
-        var cookieSales = this.cookieSales();
-        this.totalDay += cookieSales;
-        this.saleResults.push(cookieSales);
-        var newLi = document.createElement('li');
-        var newContent = document.createTextNode(this.hours[i] + ': ' + cookieSales + ' cookies');
-        newLi.appendChild(newContent);
-        var current = document.getElementById('li4');
-        document.body.insertBefore(newLi, current);
-      } else {
-        var newLiZ = document.createElement('li');
-        var newContentZ = document.createTextNode('Total: ' + this.totalDay);
-        newLiZ.appendChild(newContentZ);
-        var currentZ = document.getElementById('li4');
-        document.body.insertBefore(newLiZ, currentZ);
-        break;
-      }
-    }
-  },
+
+stores[0].cookieSales();
+
+// Store.prototype.cookiesPerHour = function() {
+//   for(var i = 0; i < 15; i++) {
+//     var cookieSales = this.cookieSales();
+//     var newLi = document.createElement('li');
+//     var newContent = document.createTextNode(cookieSales + ' cookies');
+//     newLi.appendChild(newContent);
+//     var current = document.getElementById('li1');
+//     document.body.insertBefore(newLi, current);
+//   }
+// };
+
+// for(var i = 0; i < 5; i++)
+//   stores[i].cookiesPerHour();
+
+
+Store.tableHours = function(){
```

Comment:
you should be attaching methods to the prototype, not directly to the constructor